tests/test_cms_menus.py

```diff
@@ -726,6 +839,37 @@ def test_menu_with_softroot_rendering_nested_softroot_child(self):
 
         self.assertTreeQuality(soft_root, mock_tree, 'title', 'level')
 
+    def test_draft_menu_item_atributechanges_on_published_page_menu(self):
+        """
+        Draft menu item attribute changes render changes in draft menucontent version only.
+        """
+
+        original_version = factories.MenuContentWithVersionFactory(version__state=PUBLISHED, language=self.language)
+        original_root = factories.ChildMenuItemFactory(
+            parent=original_version.root, content=self.root_pagecontent
+        )
+
+        original_child = factories.ChildMenuItemFactory(parent=original_root, content=self.aaa_pagecontent.page)
```

Comment:
Because we are testing attribute changes we need to explicitly set them on each version, this version should set them to False if we are testing them as True further down.